chainer/functions/connection/deconvolution_2d.py

```diff
@@ -325,21 +325,49 @@ def deconvolution_2d(x, W, b=None, stride=1, pad=0,
                      outsize=None, use_cudnn=True, deterministic=False):
     """Two dimensional deconvolution function.
 
-    This is an implementation of two-dimensional deconvolution.
+    This is an implementation of two-dimensional deconvolution (alse known as
+    transposed convolution).
```

Comment:
According to [the tutorial in Theano](http://deeplearning.net/software/theano/tutorial/conv_arithmetic.html#about-this-tutorial) it is a misusage to call transposed convolution deconvolution. TensorFlow also [mentions the misusage](https://github.com/tensorflow/tensorflow/blob/4c3b060b9b063a247172c939e0e1314577b581e3/tensorflow/python/ops/nn_ops.py#L46).